src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -60,9 +60,7 @@ export class HuiGenericEntityRow extends LitElement {
     // By default, we always show a pointer, since if there is no explicit configuration provided,
     // the frontend always assumes "more-info" in the action handler. We only need to hide the pointer
     // if the tap action is explicitly set to "none".
-    const pointer = !(
-      this.config.tap_action && this.config.tap_action.action === "none"
-    );
+    const pointer = hasAnyAction(this.config);
```

Comment:
Was missing a cursor when there is a hold_action, but no tap_action. In other cases we display a pointer if there is any defined action, not just tap.